api/nsadm/service.go

```diff
@@ -173,3 +175,15 @@ func (s *service) GetDataUserSecurity(ctx context.Context, tenant string) (bool,
 	}
 	return false, ErrUnauthorized
 }
+
+func (s *service) ListTokens(ctx context.Context, namespace string) (*models.Token, error) {
+	ns, _ := s.store.GetNamespace(ctx, namespace)
+	if ns != nil {
```

Comment:
Invest the logic here. This reduces the code identation and allow for easier reading of code. Something like:
```go
if ns == nil {
    return nil, ErrTokenNotFound
}
```

So rest of the code can take the assumption `ns` is valid.